compiler/src/dmd/target.d

```diff
@@ -222,7 +222,11 @@ void addPredefinedGlobalIdentifiers(const ref Target tgt)
     addCRuntimePredefinedGlobalIdent(tgt.c);
     addCppRuntimePredefinedGlobalIdent(tgt.cpp);
 
-    if (tgt.isX86_64)
+    if (tgt.isAarch64)
+    {
+        VersionCondition.addPredefinedGlobalIdent("Aarch64");
```

Comment:
Documentation mentions `AArch64`, typo?

Reply:
which documentation?

Reply:
Sorry, predefined versions, as Iain says.

Reply:
Ok. I'll fix it.

Reply:
done